.github/workflows/build-electron.yml

```diff
@@ -7,29 +7,111 @@ jobs:
-    
+
     - name: Install Node.js
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-
-    - name: Download invest binaries
+    
+    - name: Set env var for invest version
       shell: bash
-      # "-- <arg>" in run passes the arg through to the script
+      run: echo ::set-env name=INVEST_VERSION::$(jq -r .invest.version package.json)
+
+    - name: Clone natcap/invest
+      uses: actions/checkout@v2
+      with:
+        repository: natcap/invest
+        ref: refs/tags/${{env.INVEST_VERSION}}
+        path: ./invest
+
+    - name: Set up conda for non-Windows
+      uses: goanpeca/setup-miniconda@v1.1.2
+      if: matrix.os != 'windows-latest'
+      with:
+        activate-environment: invest-env  # This is also hardcoded in Pyinstaller's invest.spec
+        auto-update-conda: true
+        python-version: 3.7
+        channels: conda-forge
+
+    - name: Install python dependencies for non-Windows
+      if: matrix.os != 'windows-latest'
+      shell: bash -l {0}
+      working-directory: ./invest
+      run: |
+          conda upgrade -y pip setuptools
+          conda install toml requests
+          conda install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
+          python ./scripts/convert-requirements-to-conda-yml.py \
+                requirements.txt \
+                requirements-dev.txt > requirements-all.yml
+          conda env update --file requirements-all.yml
+
+          # Libtiff from conda defaults channel gets around issue with missing libwebp
+          conda install --yes libtiff
+
+          # The dependencies not required by natcap.invest
+          conda install flask
+
+    - name: Set up Windows python
+      if: matrix.os == 'windows-latest'
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+        architecture: x64
+
+    - name: Install python build dependencies for Windows
+      if: matrix.os == 'windows-latest'
+      working-directory: ./invest
+      shell: bash -l {0}
+      run: |
+          python -m pip install --upgrade pip nose setuptools toml twine ${{ matrix.numpy }}
+          pip install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
+
+    - name: Install python runtime dependencies for Windows
+      if: matrix.os == 'windows-latest'
+      working-directory: ./invest
+      shell: bash -l {0}
+      env:
+        PIP_EXTRA_INDEX_URL: "http://pypi.naturalcapitalproject.org/simple/"
+        PIP_TRUSTED_HOST: "pypi.naturalcapitalproject.org"
+        PIP_PREFER_BINARY: 1
+        SITE_PACKAGES: "C:/hostedtoolcache/windows/Python/3.7.8/x64/lib/site-packages"
+      run: |
+        python -m pip install -r requirements.txt -r requirements-dev.txt
+        pip install flask
+        cp ${{ env.SITE_PACKAGES }}/shapely/DLLs/geos_c.dll ${{ env.SITE_PACKAGES }}/shapely/DLLs/geos.dll
```

Comment:
Is this shapely DLL dance related to Shapely 1.7.0+ ?

Reply:
@dcdenu4 , I think it goes way back. See also invest's `ci/windows-ci-binary-install.ps1` script.

Reply:
Woah, I don't think I've ever looking in that file before. Thanks!